django_netjsongraph/base/admin.py

```diff
@@ -18,10 +20,17 @@ def __init__(self, *args, **kwargs):
 
 
 class BaseAdmin(TimeStampedEditableAdmin):
+    save_on_top = True
+
     class Media:
-        css = {'all': [static('netjsongraph/admin.css')]}
-        js = [static('netjsongraph/receive-url.js'),
-              static('netjsongraph/strategy-switcher.js')]
+        css = {'all': [static('netjsongraph/css/style.css'),
```

Comment:
`netjsongraph/css/style.css` must be moved after `netjsongraph/css/src/netjsongraph.css` because it's an override